### ISSUE TITLE
Form submission mutation

### ISSUE THREAD

**Issue by tamlyn** [elifesciences/elife-xpub#153]
Create a mutation which gets called upon submitting the final page of the submission wizard. This needs to:

- validate the full submission on the server
- move the submission to the next stage
- send a confirmation email to the submitter (see #53)

**Comment by hdrury1** [comment]
Related to #154 (?)

**Comment by tamlyn** [comment]
Related in that they both involve sending emails but my understanding is that the verification email is sent when leaving the first page of the wizard, before the user submits the final page.

**Comment by dryhten** [comment]
@hdrury1 From my understanding that is a different e-mail.

#154: sends verification e-mail when you click next after first page
this issue: sends confirmation e-mail at the end of all 4 pages

**Comment by hdrury1** [comment]
Yes! You are both right of course.

**Comment by hdrury1** [comment]
Deleted comment thread to avoid confusion.

**Comment by dryhten** [comment]
Currently blocked by #53

**Comment by diversemix** [comment]
See comment on #53 ... @hdrury1 I think in general we should avoid hierarchical tickets. What do you think?

**Comment by hdrury1** [comment]
@dryhten I don't see why this is blocked? We don't need to know at this stage exactly what we are going to send the submitter- just that we have the ability to send some dummy text for now and the ability to easily change this in the future once we have final text from the Editorial team.

**Comment by dryhten** [comment]
@hdrury1 as per the slack discussion, it might be harder to merge the code with a bad mailer config and commented out code is not the best option. I'll check tomorrow if I can get it to work with a fake config and if it does I guess we can close this after code review.

**Comment by hdrury1** [comment]
Just wanted to alert you to an update to the product requirements, relevant to this ticket.

Upon form submission, the submitter SHOULD NOT receive an email. Instead, they should be redirected to their dashboard where they will see their most recent submission listed at the top of the page with its status updated to 'Awaiting Quality Checks'.

**Comment by diversemix** [comment]
@hdrury1 / @dryhten  ... my view on this is that this should go in as-is and we raise another ticket for the change in requirement ... if you're okay with that Hannah?
